components/centraldashboard/public/components/main-page.js

```diff
@@ -0,0 +1,98 @@
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
```

Comment:
**_[nit]_** But can we do imports, and generally all objects like
```
import {PolymerElement, html} from '@polymer/polymer/polymer-element.js'
or
const {a, b} = obj
```
- No padding around imports / objects
- No semicolons

Reply:
That seems to work. I've always just stuck with the conventions from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import. Is there a reason why you explicitly want this permitted?

As an aside, we should probably add linting to the webpack configuration at some point to check these at build time.

Reply:
If you read further down you'll see most examples mention it without the padding. It's kinda like the `{` on its own line or the same line. I think the latter _(without spaces)_ tends to be more common in OSS and is generally more condense, without sacrificing readability.

It really is not a big issue. Just something I am used to seeing/using in most codebases.